### PR TITLE
CAL-30 Moved NITF Mime Type Configuration from platform-app  to imaging-app

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR adds the NITF Custom Mime Types from DDF to Alliance
#### Who is reviewing it?
@jlcsmith 
@kcwire 
@dcruver 
#### How should this be tested?
Build Alliance and Observe the NITF Custom Mime Type
#### Any background context you want to provide?
Since the NITF Input transformer has been moved to Alliance, there is no longer the need for this configuration in DDF.
#### What are the relevant tickets?
https://codice.atlassian.net/browse/CAL-30
https://codice.atlassian.net/browse/DDF-2038
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

…

### CODE REVIEW

**Comment by dcruver** [comment]
lgtm

File: catalog/imaging/catalog-imaging-app/src/main/resources/features.xml [hunk 1]
```diff
@@ -20,5 +20,6 @@
              description="The Alliance Imaging Application provides support for ingesting and searching for NITF products.  ::Alliance Imaging">
         <feature prerequisite="true">catalog-app</feature>
         <bundle>mvn:com.connexta.alliance.imaging/alliance-imaging-transformer/${project.version}</bundle>
+        <configfile finalname="/etc/DDF_Custom_Mime_Type_Resolver.nitf.config">mvn:com.connexta.alliance.imaging/alliance-imaging-app/${project.version}/config/nitf</configfile>
```
Comment (jlcsmith) [inline catalog/imaging/catalog-imaging-app/src/main/resources/features.xml]: The `DDF_Custom_Mime_Type_Resolver.nitf.config` should be in the `alliance-imaging-transformer` project.

**Comment by jlcsmith** [comment]
I'm getting a build failure related to the nitf resolver config file.  

`[ERROR] Failed to execute goal org.apache.karaf.tooling:karaf-maven-plugin:4.0.4:kar (kar) on project alliance-imaging-app: Failed to create archive: Could not transfer artifact com.connexta.alliance.imaging:alliance-imaging-app:config:nitf:0.1-SNAPSHOT from/to forgerock-staging-repository (http://maven.forgerock.org/repo/releases): Failed to transfer file: http://maven.forgerock.org/repo/releases/com/connexta/alliance/imaging/alliance-imaging-app/0.1-SNAPSHOT/alliance-imaging-app-0.1-SNAPSHOT-nitf.config. Return code is: 409 , ReasonPhrase:Conflict.
[ERROR] com.connexta.alliance.imaging:alliance-imaging-app:config:0.1-SNAPSHOT`

**Comment by bdeining** [comment]
Closing PR based on discussions.

**Pull request closed without merging**